ddl/placement_policy_test.go

```diff
@@ -1011,9 +1011,15 @@ func (s *testDBSuite6) TestDatabasePlacement(c *C) {
 func (s *testDBSuite6) TestDropDatabaseGCPlacement(c *C) {
 	clearAllBundles(c)
 	failpoint.Enable("github.com/pingcap/tidb/store/gcworker/ignoreDeleteRangeFailed", `return`)
-	defer func() {
+	defer func(originGC bool) {
 		failpoint.Disable("github.com/pingcap/tidb/store/gcworker/ignoreDeleteRangeFailed")
-	}()
+		if originGC {
+			ddl.EmulatorGCEnable()
+		} else {
+			ddl.EmulatorGCDisable()
+		}
+	}(ddl.IsEmulatorGCEnable())
+	ddl.EmulatorGCDisable()
```

Comment:
why these GC test without the EmulatorGC?

Reply:
EmulatorGC will delete keys (but not delete placement rules) from read all gc jobs from `gc_delete_range` and then move these jobs to `gc_delete_range_done` .  So when we `gcWorker.DeleteRanges` manually after it, placement rules will not be gc because table `gc_delete_range` is empty